python/pyspark/sql/types.py

```diff
@@ -188,6 +199,8 @@ class IntegerType(PrimitiveType):
 
     The data type representing int values.
     """
+    def simpleString(self):
+        return 'int'
```

Comment:
Not easy, then we may need another different implementation of DataTypes, we could do the refactor later.